data/wiki/hello-world.md

```diff
@@ -142,9 +108,10 @@ Congratulations! You've just built and run your very first unikernel!
 
 ### Step 1: Hello World!
 
-Most programs will depend on some system devices which they use to interact with
-the environment. In this section, we illustrate how to define unikernels that
-depend on such devices. 
+Except for the noop example above, all unikernels have at least one job.
+Furthermore, most programs will depend on some system devices which they use to
```

Comment:
It may also do some good to give some very basic examples of devices

Reply:
I was thinking to write something like "the rest of this tutorial will cover examples using timer devices, key-value stores, block devices and network interfaces". I then struggle with the sentence following "In this section, we illustrate how to define unikernels with jobs that depend on such devices". Do you have a suggestion how to word it?

Reply:
how about?
```
...Most jobs will depend on some system devices which they use to interact with the environment. This tutorial will cover some examples using timer devices, key-value stores, block devices and network interfaces.
In this section, we illustrate how to define unikernels with jobs that depend on a timer device.
```